pakyow-core/lib/pakyow/core/routing/route.rb

```diff
@@ -59,19 +59,6 @@ def populated_path(path_to_self, **params)
         }.join("/"))
       end
 
-      def freeze
-        hooks.each do |_, hooks_arr|
```

Comment:
Right you are. I just pushed an addition that refines Hash. I'm freezing keys and values. Do you think that is the right behavior? Or should it only be values?